src/components/List/List.stories.js

```diff
@@ -1,13 +1,42 @@
-/* @flow */
```

Comment:
I think it is really useful to have flow here, then we are getting notified when changing the interface for the component.